apps/web/app/[lang]/studies/[id]/components/StudyPreviewContent.tsx

```diff
@@ -0,0 +1,124 @@
+import { P2, Card, H4, Button, HorizontalDivider } from 'ui/core';
+import { useTranslations } from 'intl';
+import { ImageWithFallback } from '@components/utils/ImageWithFallback';
+import { AgeToText, GenderToText } from '../helpers';
+import { StudyPreviewContentProps } from '../Study.types';
+
+export const StudyPreviewContent: React.FC<StudyPreviewContentProps> = ({
+  study,
+}) => {
+  const t = useTranslations('studies.studyDetail');
+  const tActions = useTranslations('actions');
+  const tStudyTypes = useTranslations('studies.studyDetail.types');
+  const ageRequirements = AgeToText(
+    study.eligibility.minimum_age,
+    study.eligibility.maximum_age,
+  );
+
+  return (
+    <Card className="w-full max-h-min flex flex-col items-center justify-center !m-0 !p-4 gap-6">
```

Comment:
```suggestion
    <Card className="w-full flex flex-col justify-center !m-0 !p-4 gap-6">
```

Remove unneeded classes